sharding-and-replication.md

```diff
@@ -0,0 +1,171 @@
+# Sharding and Replication
+
+This is a guide how to setup sharding and replication for Jaeger data.
+This guide uses [clickhouse-operator](https://github.com/Altinity/clickhouse-operator) to deploy
+the storage.
+
```

Comment:
I would suggest putting a recommendation to use latest LTS Clickhouse version (21.3 currently).

Reply:
New is always better :) Are there any particular features that we might use?